Framework/Utils/include/DPLUtils/Utils.h

```diff
@@ -18,16 +18,14 @@
 
 #include "Framework/DataProcessorSpec.h"
 #include <functional>
+#include "MemoryResources/MemoryResources.h"
 
 namespace o2f = o2::framework;
 
 namespace o2
 {
 namespace workflows
 {
-//TODO: this is to make DPLmerger compile, but this code is (and always was) horribly broken - please remove.
-inline void freefn(void* data, void* /*hint*/) { delete static_cast<char*>(data); };
```

Comment:
Difficult to judge for me, but if this is broken and should be removed, and it compiles after removal, then it is probably the way to go.

Reply:
probably, a remnant after removal of the Stack::freefn long time ago.